src/ontology/mondo-edit.obo

```diff
@@ -313201,6 +313201,7 @@ xref: SCTID:315345002 {source="EFO:0004610", source="MONDO:equivalentTo"}
 xref: UMLS:C0242488 {source="Orphanet:178320", source="Orphanet:178320/e", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0005087 {source="Orphanet:178320"} ! respiratory system disorder
 is_a: MONDO:0005275 {source="EFO:0004610", source="MESH:D055371/inferred"} ! lung disorder
+is_a: MONDO:0021178 {source="https://orcid.org/0000-0001-5493-2602"} ! injury
```

Comment:
```suggestion
is_a: MONDO:0021178 {source="https://orcid.org/0000-0001-5493-2602"} ! injury of the sould
```

Reply:
@matentzn is this a real change that you want?